src/components/icon/icon.spec.js

```diff
@@ -514,29 +510,39 @@ describe('MdIcon service', function() {
     });
 
     describe('icon group is not found', function() {
-      it('should log Error', function() {
-        var msg;
-        try {
-          $mdIcon('notfound:someIcon')
-            .catch(function(error) {
-              msg = error.data;
-            });
+      it('should log Error and reject', inject(function($log) {
+        var errorMessage = 'Cannot GET notfoundgroup.svg';
+        var caughtRejection = false;
+
+        $mdIcon('notfound:someIcon')
+          .catch(function(error) {
+            expect(error.data).toBe(errorMessage);
+            caughtRejection = true;
+          });
 
-          $httpBackend.flush();
-        } finally {
-          expect(msg).toEqual('Cannot GET notfoundgroup.svg');
-        }
-      });
+        $httpBackend.flush();
+
+        expect(caughtRejection).toBe(true);
+        expect($log.warn.logs[0]).toEqual([errorMessage]);
+      }));
     });
 
     describe('icon is not found', function() {
-      it('should not throw Error', function() {
-        expect(function(){
-          $mdIcon('notfound');
+      it('should log Error and reject', inject(function($log) {
```

Comment:
Thanks for the docs fix!

So, I guess my question is: what happens if they just type `$mdIcon('notfound')`? You're saying that pre 1.6, it will work identically as before? It's just that in 1.6 they will get a PUR?

If so, then I think this is good.